bigtable/client/client_options.h

```diff
@@ -15,6 +15,7 @@
 #ifndef GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_CLIENT_OPTIONS_H_
 #define GOOGLE_CLOUD_CPP_BIGTABLE_CLIENT_CLIENT_OPTIONS_H_
 
+#include "absl/base/internal/throw_delegate.h"
```

Comment:
We should never directly include headers from `absl/*/internal/*`, please remove.